crates/editor/src/editor.rs

```diff
@@ -3050,6 +3052,7 @@ impl Editor {
             self.refresh_document_highlights(cx);
             self.refresh_selected_text_highlights(false, window, cx);
             refresh_matching_bracket_highlights(self, window, cx);
+            refresh_rainbow_bracket_highlights(self, window, cx);
```

Comment:
Rainbow brackets don't change on selection change, and so re-computation shouldn't be done here.